lib/plugins/create/templates/aws-nodejs-typescript/handler.ts

```diff
@@ -1,4 +1,5 @@
 import { APIGatewayEvent, Callback, Context, Handler } from 'aws-lambda';
+import "source-map-support/register";
```

Comment:
Personally I don't like having to remember to register every file.
When using webpack I prefer [this approach](https://github.com/serverless/serverless/compare/master...franciscocpg:add-source-map-support).

Reply:
+1, should definitely be in the Webpack configuration instead of the handler.

Reply:
I vote of your approach. (Didn't know it was possible)

Reply:
@sime 
Are you going to apply this approach in this PR?

Reply:
@franciscocpg Happy to! But since you have the code already written you can submit the PR.

Reply:
@sime 
Since this PR is already open, would you mind merging the code into this PR?

I think running this inside your `typescript-source-maps` branch would be enough:
```sh
git remote add franciscocpg https://github.com/franciscocpg/serverless.git
git fetch franciscocpg
git merge franciscocpg/add-source-map-support
```

Reply:
@franciscocpg Thanks for the step by step !

Done :)

Reply:
:+1: 
@sime now that you've merged it you can safely remove `import "source-map-support/register";` from this file.